include/z64save.h

```diff
@@ -4,6 +4,25 @@
 #include "ultra64.h"
 #include "z64math.h"
 
+typedef enum {
+    /*  0 */ HUD_VISIBILITY_IDLE,
+    /*  1 */ HUD_VISIBILITY_NONE,
+    /*  2 */ HUD_VISIBILITY_NONE_ALT, // Identical to HUD_VISIBILITY_NONE
+    /*  3 */ HUD_VISIBILITY_HEARTS_WITH_OVERWRITE, // Uses Interface_UpdateButtonAlphas so gives the opportunity to dim only disabled buttons
```

Comment:
Could you explain the reasoning behind this comment and its other iterations? I'm not following what "dim only disabled buttons" refers to in the code

Reply:
I think it makes sense yeah, tho it's unfortunate that we have to make this distinction in the first place. An argument could be made that the` _WITH_OVERWRITE` modes should be the "default" ones, and the alternative ones named `*_FORCE` or whatever, but I'm not sure that'd be better overall.

Also the comments are definitely easier to understand now 👍

Reply:
At the very least, `OVERWRITE` was based on a previous name of `forceRisingButtonAlphas`, but `overwrite` -> `force` was a change we made along the way, so I'm fine with `OVERWRITE` -> `FORCE` just for consistency